src/components/components/CandidateProfileCard/CandidateProfileCard.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./CandidateProfileCard.scss";
+import CandidateImage from "./assets/candidateImage.svg";
```

Comment:
i think we need to send candidate Image through the props too

Reply:
makes sense, thank you. updated the code